ledgers/ccf/pdo/ledgers/ccf/scripts/set_contract_enclave_check_attestation_flag.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import http
+import sys
+
+from loguru import logger as LOG
+
+from pdo.ledgers.ccf.common import parse_common_arguments
+
+
+# -----------------------------------------------------------------
+def set_contract_enclave_check_attestation_flag(client, options):
+
+    params = {}
+    params['check_attestation'] = options.check_attestation
+
+    r = client.post("/app/set_contract_enclave_check_attestatation_flag", params)
+    if r.status_code != http.HTTPStatus.OK.value:
+        LOG.error('failed to set contract enclave check-attestation flag: {}, code: {}'.format(
+            r.body, r.status_code))
+        sys.exit(-1)
+
+# -----------------------------------------------------------------
+def Main() :
+
+    (_, unprocessed_args, member_client) = parse_common_arguments(
+        sys.argv[1:], 'Fetch the ledger authority key from a CCF server', True)
+
+    # Parse the arguments that are unique to the script
+    parser = argparse.ArgumentParser(description='Set contract enclave attestation check flag')
+    parser.add_argument('--check-attestation', default=False, help="enable attestation verification", action='store_true')
+
```

Comment:
there should be both positive (check-attestation) and negative (no-check-attestation) flags. they should be declared required and should be mutually exclusive.

Reply:
thanks, added a mutually exclusive bool flag